### DISCUSSION TITLE
How users can get email notification?

### DISCUSSION THREAD

**Discussion by syeda-git** [giscus/giscus#1258] [Q&A]
Hi,
I am the admin of the Github repo. I am notified by email whenever someone posts a comment in Giscus. However, the users who posted comments are not notified for their posts and responses. They have set up Email ID in Settings >Email Notifications, Settings >Notifications, and are Watching Discussions.

Do they have to do something more to get notified?

**Comment by laymonage** [comment]
It's nothing specific to giscus, so check your GitHub settings e.g.

https://github.com/settings/notifications

<img width="331" alt="image" src="https://github.com/giscus/giscus/assets/6379424/66f8958a-d6f3-4e8b-b537-985f53220247">

<img width="331" alt="image" src="https://github.com/giscus/giscus/assets/6379424/7c650a93-c37e-4d83-9334-0278bccb17f6">

**Reply by syeda-git** [reply]
Thanks @laymonage . Those settings were already configured. Not sure why the notifications are not sent. Let me check in Github forums.
